wordle-web/tests/wordList.test.ts

```diff
@@ -0,0 +1,142 @@
+import { describe, it, test, expect } from 'vitest';
+import { Game } from '~/scripts/game';
+import { myWordList } from '~/scripts/wordList';
+import { ValidWordsUtils } from '~/scripts/validWordsUtils';
+
+const wordList = [
+  'AARGH',
+  'ABACA',
+  'HELLO',
+  'THINK',
+  'PEACH',
+  'BEACH',
+  'WHICH',
+];
+
+const wordList2 = ['ABACA', 'ABODE', 'TAROT'];
+
+describe('validWords_enteredLettersIn_returnsCorrectWordAndLength', () => {
+  it('enteredA_returnsFirstTwoWords', () => {
+    let game = new Game();
+    game.secretWord = 'HELLO';
+    game.addLetter('A');
```

Comment:
Great test, the only thing I might recommend is using the whole word List rather than a short one, it may help show any bugs in the valid Word method if there is any.

Reply:
True. I chose to do this because I can simulate anything with a smaller list. Sifting through the larger list to find what to look for is too time consuming for me. This way, I could spend more time thinking of edge cases! :)